src/components/ProjectList/ProjectItem.js

```diff
@@ -1,47 +1,46 @@
-// TODO: add flow annotations
-/* eslint-disable */
+/* @flow */
+
 
 import React, { Component } from 'react'
-import _ from 'lodash'
+import moment from 'moment'
 import Col from 'react-bootstrap/lib/Col'
 import Row from 'react-bootstrap/lib/Row'
 import Divider from 'material-ui/Divider'
 import { ListItem } from 'material-ui/List'
+import type { ProjectType } from 'ducks/projects'
 
 const subHeader = text => (
   <div
-    style={{ color: '#8c8c8c', fontSize: '13px' }}
+    style={{ color: '#8c8c8c', fontSize: '14px' }}
   >
     {text}
   </div>
 )
 
-export type Props = {
-  item: Object,
+type Props = {
   clickHandler: Function,
-  childrenProp: string,
-  primaryTextProp: string,
-  secondaryTextProp?: string,
-  inset?: // updated: PropTypes.string,
-  // owner: PropTypes.string,
-  number,
-  valueProp: string,
+  inset: number,
+  project: ProjectType,
 }
 
 class ProjectItem extends Component {
   constructor(props: Props) {
     super(props)
-
     this.state = {
       open: false,
       followed: false,
     }
-    this.toggleOpen = this.toggleOpen.bind(this)
-    this.toggleFollow = this.toggleFollow.bind(this)
+    this.touchTapEventHandler = this.touchTapEventHandler.bind(this)
```

Comment:
Flow probably complains about to things. You need to add a ` /* eslint-disable react/sort-comp */ ` and `; (this:any).touchTapEventHandler = this.touchTapEventHandler.bind(this)`